python/ray/job_config.py

```diff
@@ -135,11 +134,13 @@ def set_default_actor_lifetime(self, default_actor_lifetime: str) -> None:
         Args:
             default_actor_lifetime: The default actor lifetime to set.
         """
+        import ray.core.generated.common_pb2 as common_pb2
```

Comment:
For the protobuf, any special reason we import inside a method?

Reply:
This is needed so the docs build is working (some stuff gets mocked there and if this is top level, that prevents things from working). Now that I'm thinking about this, let me actually wrap this use in Cython -- then we can get rid of this import and also this will be needed anyways going forward when we also want to get rid of the Python protobuf dependency as well :)